src/utils/LoginValidation.jsx

```diff
@@ -1,10 +1,10 @@
-export const validate = (fieldName, value) => {
+export const validate = (fieldName, value, stateObj) => {
   let error = '';
   switch (fieldName) {
     case 'email':
       if (!value) {
         error = 'Email address is required';
-      } else if (!/\S+@\S+\.\S+/.test(value)) {
+      } else if (!/\S+\w*@gmail\.+com\b/.test(value)) {
```

Comment:
Move this into method

isValidEmail = (value: string) => !/\S+\w*@gmail\.+com\b/.test(value)